src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.270.3.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Browser agent
+releaseDate: "2024-10-31"
+version: 1.270.3
+features: []
+bugs: ["Ensure all lazy loaded modules issue warning instead of errors"]
+security: []
+---
+
+## v1.270.3
+
+### Bug fixes
+
+#### Ensure all lazy loaded modules issue warning instead of errors
+Lazy loaded modules that are blocked from loading by third party tools such as adblocking mechanisms will issue a warning to console instead of throwing an error.
```

Comment:
```suggestion
Lazy loaded modules that are blocked from loading by third-party tools (such as ad blocking mechanisms) will issue a warning to console instead of throwing an error.
```